### PR TITLE
Vision nncf quantization and prunign examples

### PR DESCRIPTION
**Pull request by Akash-guna**

### CODE REVIEW

File: examples/vision/mmrazor_pruning/example.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+logs/
+	└── log.log
+```
+	
+####  7. Results
+
+We use MAP and Flops to assess the result produced. The results are as follows:
+	|Model | MAP | Latency|
+   |Yolox-tiny|32.0|0.3515 ms|
+   |YoloX-tiny pruned| 29.6| 0.3170 ms
```
Comment (shwu-nyunai) [inline examples/vision/mmrazor_pruning/example.md:71]: 1. (nits) we can use 'tree <path>' to get the tree structure of the output dir for a cleaner look
2. the results table isn't correctly formatted (lets use https://www.tablesgenerator.com if required) 
image

File: examples/vision/nncf_quantization/example.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+``CONFIG Path`` Config path is the edited yaml used for defining the hyperparameters of the job. 
+
+After the job is completed the folder structure will be as follows
+```
+jobs/
+	└──  mds.xml
+	└── mds.bin
+logs/
+	└── log.log
+```
```
Comment (shwu-nyunai) [inline examples/vision/nncf_quantization/example.md:63]: (nits) same as above.

File: examples/vision/nncf_quantization/example.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+	
+####  7. Results
+
+The model post finetuning on cifar-10 for 10 epochs had validation 67.4 % validation accuracy and after quantization had an accuracy of 67.2 %. To benchmark the reduced latency of the model use [NNCF Benchmarking tool](https://docs.openvino.ai/2024/learn-openvino/openvino-samples/benchmark-tool.html).
```
Comment (shwu-nyunai) [inline examples/vision/nncf_quantization/example.md:67]: can we have a table here for quick comparison? or is it not needed?

**Review by shwu-nyunai — CHANGES_REQUESTED** [review]

File: examples/vision/nncf_quantization/example.md (hunk 3 above)
Comment (Akash-guna) [reply examples/vision/nncf_quantization/example.md:67]: yeah ig will add a small table here

Comment (Akash-guna) [reply examples/vision/nncf_quantization/example.md:67]: hey there is just 2 results here is a table necessary here

Comment (Akash-guna) [reply examples/vision/nncf_quantization/example.md]: added latency and the table

**Comment by shwu-nyunai** [comment]
Also, let's rename the files as README.md to open them in rich format, by default, right inside the parent directories.

**Review by shwu-nyunai — APPROVED** [review]
lgtm
